### ISSUE TITLE
Improve "Submit a Work" Button

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/Pilcrow#1914]
There's not enough whitespace above the "Submit a Work" button. The language on this button also violates the established controlled vocabulary by referring to a "submission" as a "work".

Add whitespace above this button and change its text.

**Comment by sattler9** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pilcrow-Development-5ff39ba363391e0019713c87/issues/MESH-Research/Pilcrow/1914?planning-poker) with Zenhub @gmeben @vilagshe @wreality

**Comment by gmeben** [comment]
This ended up being slightly more tricky than I thought. I didn't realize this button appears twice in the application. Also, the text "Submit a Work" appears 3 times in total throughout the application, all sourced from the same line of text in the internationalization file:

* On a publication's home page as a button
    * I chose to change the text to "Submit New Submission" here
* On the submissions page as a button
    * I chose to change the text to "Submit New Submission" here
* On the submission creation page as a page title
    * I chose to change the text to "Create Submission" here

I only saw a lack of whitespace above the button as it appears on a publication's home page, so I added the extra whitespace there.
